src/ServiceControlInstaller.Packaging/ServiceControlInstaller.Packaging.csproj

```diff
@@ -16,7 +16,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Packaging.ServiceControl.Monitoring" Version="2.0.1" PrivateAssets="All" />
+    <PackageReference Include="Packaging.ServiceControl.Monitoring" Version="2.0.2" PrivateAssets="All" />
```

Comment:
Why is this change bundled into the PR?

Reply:
Monitorid had the same issue and needs to be released as well so that’s why this is here?

Reply:
So 3.3.4 will contain both the SC fix and the monitoring fix

Reply:
:+1: